src/main/java/net/minestom/server/entity/Metadata.java

```diff
@@ -31,7 +32,7 @@ public static Entry<Integer> VarInt(int value) {
     }
 
     public static Entry<Long> Long(long value) {
-        return new MetadataImpl.EntryImpl<>(TYPE_LONG, value, NetworkBuffer.LONG);
+        return new MetadataImpl.EntryImpl<>(TYPE_LONG, value, NetworkBuffer.VAR_LONG);
```

Comment:
This has overflowed from 1.19.3 but `#Long` should be renamed to `#VarLong` and `TYPE_LONG` to `TYPE_VAR_LONG`